src/Model/ArticleInterface.php

```diff
@@ -158,11 +158,11 @@ public function setSubtitle($subtitle);
     public function getSubtitle();
 
     /**
-     * @param array $tags
+     * @param iterable $tags
```

Comment:
This should either be removed or improved by precising the type of tag with the `TypeOfTheTag[]` notation.